debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbChangeEventSourceFactory.java

```diff
@@ -64,7 +68,7 @@ public SnapshotChangeEventSource<MongoDbPartition, MongoDbOffsetContext> getSnap
                 snapshotProgressListener,
                 errorHandler,
                 notificationService,
-                snapshotterService);
+                snapshotterService, this.beanRegistry);
```

Comment:
```suggestion
                snapshotterService, 
                this.beanRegistry);
```
I know we aren't uniform about this behavior, but as a side note, I'd prefer if we have blocks of code that use the style here where arguments are per line, adding new arguments should adhere to that same style. I think I may have caught a few others, but I could have missed a few.